angio-frontend/src/modules/user/components/AccountActivationForm.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+    <v-card>
```

Comment:
Add paddings like the `BaseSingleFormContainer`

Reply:
Done